packages/block-library/src/columns/deprecated.js

```diff
@@ -66,21 +66,21 @@ export default [
 			) );
 		},
 		migrate( attributes, innerBlocks ) {
-			const columns = innerBlocks.reduce( ( result, innerBlock ) => {
+			const columns = innerBlocks.reduce( ( resultAccumulator, innerBlock ) => {
```

Comment:
It seems like `resultAccumulator` could be simplified to just `accumulator`. They `result` prefix doesn't sound like it brings additional context here.